### DISCUSSION TITLE
how to use broadcast-notifications

### DISCUSSION THREAD

**Discussion by cnpap** [filamentphp/filament#7371] [Help]
i used ably for my app. it's work.

### what i expect?
run the code & filament page got some message

```
<?php

namespace App\Console\Commands;

use Filament\Notifications\Notification;
use Illuminate\Console\Command;

class TestSendNtfCommand extends Command
{
    protected $signature = 'test:send-ntf';

    protected $description = 'Command description';

    public function handle(): void
    {
        $this->info('test msg');
        Notification::make()
            ->title('Saved successfully')
            ->success()
            ->send();
    }
}
```

# my current situation

according to the official documention, i should be use `echo`.
![image](https://github.com/filamentphp/filament/assets/39842393/45c9b36f-8cd8-4272-a924-de739eb0e6ce)

```
// app.js
import './bootstrap';

// bootstrap.js

window.Pusher = Pusher;

window.Echo = new Echo({
    broadcaster: 'pusher',
    key: import.meta.env.VITE_ABLY_PUBLIC_KEY,
    wsHost: 'realtime-pusher.ably.io',
    wsPort: 443,
    disableStats: true,
    encrypted: true,
    cluster: import.meta.env.VITE_ABLY_CLUSTER
});
```

in my react page, a websocket link is generate.
but this is not show in the filament  page. 

i realized that my app.js was not loaded by filament.
i trying to change app.blade.php file content or move it. the app never changes anything.

![image](https://github.com/filamentphp/filament/assets/39842393/bca6152e-9282-40e4-9fd0-8ca8f893009e)

plase leave any information tha would help with this, thanks.

**Comment by danharrin** [comment]
Exactly the app.js is not loaded by Filament, which is why you should store your Echo config in the Filament config file. We load it for you!

**Reply by cnpap** [reply]
thanks 😉
echo is installed in npm, how do I load it

**Reply by danharrin** [reply]
You dont need to, we do it

**Reply by cnpap** [reply]
I didn't find how to configure it within the official website, has it not been updated yet.

**Reply by danharrin** [reply]
Have you published config/filament.php

**Reply by waqarpixel** [reply]
hi @danharrin  how can i register app.js in filament v3 ?
